app/jobs/fca_import_job.rb

```diff
@@ -76,7 +83,7 @@ def parse_error(list)
 
   def notification_text(outcomes)
     if import_successful?(outcomes)
-      "The FCA data have been loaded into RAD. Visit #{admin_lookup_fca_import_index_url} to confirm that the data looks ok" # rubocop:disable all
+      "The FCA data have been loaded into RAD. Visit #{admin_lookup_fca_import_index_url} to confirm that the data looks ok"
```

Comment:
should probably fix this rather than disabling with`rubocop:disable Metrics/LineLength`

Reply:
You should be able to break this up onto multiple lines without breaking anything I think by using `\`

https://github.com/moneyadviceservice/development-standards/tree/master/coding-styles/ruby#2.4

Reply:
I should make clear that the above link is for reference and the development-standards repo is still an unreviewed WIP project, so don't take everything written there as gospel. yet ;)